app/workers/migrate_shoutouts_worker.rb

```diff
@@ -0,0 +1,31 @@
+class MigrateShoutoutsWorker
```

Comment:
This should only be run once, correct? Maybe put in a TODO to have it removed later (and also add a story for this)

Reply:
@martafedori @alosikolesik please create story for this @ @ @